### PR TITLE
Bug 1479458 - Fixing broadcast of updated data from cache.

### PR DESCRIPTION
**Pull request by ScottDowne**
To test.

1. Checkout aab3771a50215d97ed4f34c7056f46a0f943a822 which is the pr before the patch that caused the bug.

2. Open about:home enough to generate some spocs and a preload and a cache.

3. Checkout my branch.

4. Go to about:home

5. Should see rec.

If you want to see the initial bug reproduced, instead of checking out my branch on step 3, checkout  be0ccddd3e6f7d7adffa55b0440f8a1104286ca5 and do everything else the same, and then on 5 should see no recs.

### CODE REVIEW

**Comment by ScottDowne** [comment]
I'm not sure how to read taskcluster or sure what it's actually testing. Think I need help in fixing it :)

**Comment by piatra** [comment]
Issue running eslint not with your code. I've merged a PR that removes linting from our taskcluster CI (since it's covered by Travis) and retriggered your build for another test.

**Comment by piatra** [comment]
oh. @ScottDowne you will need to pull the latest changes and update your PR otherwise you're still triggering the eslint check.

**Comment by ScottDowne** [comment]
@piatra got it, thanks, updating ow.

File: lib/TopStoriesFeed.jsm [hunk 1]
```diff
@@ -57,17 +55,14 @@ this.TopStoriesFeed = class TopStoriesFeed {
...
       if (this.storiesLastUpdated === 0) {
-        shouldBroadcast = true;
         await this.fetchStories();
       }
       if (this.topicsLastUpdated === 0) {
-        shouldBroadcast = true;
         await this.fetchTopics();
       }
-      this.doContentUpdate(shouldBroadcast);
+      this.doContentUpdate(true);
```
Comment (ScottDowne) [inline lib/TopStoriesFeed.jsm]: Essentially, this is how the old code worked, and I misinterpreted it when I was moving some stuff around.

**Pull request merged**